libr/core/file.c

```diff
@@ -561,7 +567,9 @@ R_API int r_core_bin_load(RCore *r, const char *filenameuri, ut64 baddr) {
 
 R_API RIOMap *r_core_file_get_next_map (RCore *core, RCoreFile * fh, int mode, ut64 loadaddr) {
 	const char *loadmethod = r_config_get (core->config, "file.loadmethod");
+	if (!loadmethod) return NULL;
```

Comment:
put them in a single line, it is better to define variables at the begining of the stackframe